decidim-proposals/app/controllers/decidim/proposals/proposal_votes_controller.rb

```diff
@@ -20,6 +29,16 @@ def create
       def check_current_settings!
         raise "This setting is not enabled for this step" unless current_settings.votes_enabled?
```

Comment:
I know this isn't related to this PR but it would be great if you could also return a valid HTTP response instead of raising.